crates/m3/src/gadgets/hash/keccak/mod.rs

```diff
@@ -165,13 +202,23 @@ impl Keccakf {
 						index.get_as(self.batches[0].state_in[(x, y)])?;
 					let batch_2_state_out: std::cell::Ref<'_, [u64]> =
 						index.get_as(self.batches[2].state_out[(x, y)])?;
+
 					for track in 0..TRACKS_PER_BATCH - 1 {
 						next_state_in[TRACKS_PER_BATCH * k + track] =
 							batch_0_state_in[TRACKS_PER_BATCH * k + track + 1];
 						assert_eq!(
 							next_state_in[TRACKS_PER_BATCH * k + track],
 							batch_2_state_out[TRACKS_PER_BATCH * k + track]
 						);
+
+						//Populating the packed and selected input and output columns.
```

Comment:
This and other comments:

```suggestion
						// Populating the packed and selected input and output columns.
```